src/IECoreMaya/SceneShapeSubSceneOverride.cpp

```diff
@@ -110,6 +110,39 @@ T parallelMaxElement( const std::vector<T> &data )
 	return maxValue;
 }
 
+bool isVisible( MFnDagNode &dagNodeFn, int instanceId )
+{
+	// figure out visibility of current node before recursively going through the parents
+	MStatus status;
+	MPlug visPlug = dagNodeFn.findPlug("visibility", &status);
+	bool visible;
+	if( MStatus::kFailure == status )
+	{
+		return false;
```

Comment:
isn't this saying the thing is not visible just because it doesn't have a visibility plug? wouldn't we want it visible unless we can prove it should not be?

Reply:
Actually, should we be checking the visibility plug ourselves anyways? I'd have thought there was a MFnDagPath (or similar) method to do all the traversing for us (and tack render layers, etc into account).